### ISSUE TITLE
checkout_index_index.xml wrongly enforces a specific page layout

### ISSUE THREAD

**Issue by wpalmer** [amzn/amazon-payments-magento-2-plugin#556]
A very similar issue was reported as #32. The problem here is that the "fix" was incorrect: rather than having Amazon Payments *not* specify a page layout, the layout was instead "updated" to the new default value. This prevented the Amazon Payments module from changing the default, if it was left as the default, but if the theme used a different page layout (for example, due to being created with the 1column layout in mind), this would instead override the "user"-specified layout.

The fix is simple: Instead of specifying a different layout, the "real" fix for #32 was to remove the `layout="..."` part of the `<page ...>` tag entirely.

ie, instead of: 
```
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="checkout" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

it should read:
```
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

Submitting this as a bug rather than a pull-request, for now, as I have not verified whether the amazon payments module currently requires any aspect of the `"checkout"` layout, for example in its styling.

As a work-around, if encountering this issue one can override the layout within one's theme, eg:

```
new file mode 100644
index 000000000..57d152be3
--- /dev/null
+++ b/app/design/frontend/MyNamespace/mytheme/Amazon_Payment/layout/override/base/checkout_index_index.xml
@@ -0,0 +1,3 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+</page>
```

#### What I expected

checkout_index_index.xml should not define a layout

#### What happened instead

checkout_index_index.xml specifies a layout, meaning any "custom" layout on the checkout page, is ignored / replaced

#### Steps to reproduce the issue

 1. define a custom layout for the checkout page in your skin's checkout_index_index.xml layout override file (eg, 1column)
 2. because the amazon payments module is loaded "after" the Magento_Checkout module, its layout override takes precedence
 3. observe that your layout override is not used, and instead the value specified by amazon payments is takes precedence.

#### Your setup

* Magento version: 2.3.2
* Amazon Pay Extension Version: 3.2.9
* Magento Edition: Community
* JavaScript Console: (irrelevant)
* Shop URL : (irrelevant)

**Comment by tarishah** [comment]
Thanks @wpalmer for reporting this. We will take a look at your recommendation and get back.

**Comment by christianzichichi** [comment]
Hi @wpalmer,

We apologize for the delay here. A fix for this issue has been included in version 3.4.1 of our plugin. this plugin version will be included in one of the future Magento 2 releases. patch instructions to upgrade and try it out in the current Magento release (2.3.4) and in earlier ones will be released very soon here: https://github.com/amzn/amazon-payments-magento-2-plugin/blob/master/PATCH_INSTRUCTIONS.MD

thanks
regards

Christian
